kura/org.eclipse.kura.wire.provider/src/main/java/org/eclipse/kura/internal/wire/WireServiceImpl.java

```diff
@@ -360,14 +363,14 @@ public synchronized void unbindWireHelperService(final WireHelperService wireHel
      *            the properties
      */
     public synchronized void updated(final Map<String, Object> properties) {
-        s_logger.debug(s_message.updatingWireService() + properties);
+        logger.debug(message.updatingWireService() + properties);
         this.extractProperties(properties);
         try {
             this.createWires();
         } catch (final KuraException e) {
-            s_logger.error(ThrowableUtil.stackTraceAsString(e));
+            logger.error(ThrowableUtil.stackTraceAsString(e));
```

Comment:
Same here as the passed arguments to exceptions. The logger can handle stack traces by itself and there is not need to pre-render this.